--- xdp-tools-1.4.1/configure.orig	2023-10-20 18:08:05.000000000 +0200
+++ xdp-tools-1.4.1/configure	2023-12-26 11:28:22.685725521 +0100
@@ -112,7 +112,7 @@ check_toolchain()
         exit 1
     fi
 
-    if command -v $BPFTOOL &>/dev/null && $BPFTOOL gen help 2>&1 | grep 'gen skeleton.*name' > /dev/null; then
+    if command -v $BPFTOOL >/dev/null 2>&1 && $BPFTOOL gen help 2>&1 | grep 'gen skeleton.*name' > /dev/null; then
         bpftool_version=$($BPFTOOL version | head -n 1)
 	echo "using $bpftool_version"
     else
